.idea/runConfigurations.xml

```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please revert these changes. This file should not be deleted. 
Once done, please open the `Files changed` section on your PR and make sure this file is not visible.